kitsune/wiki/tests/test_parser.py

```diff
@@ -297,7 +296,7 @@ def test_build_template_params_named_anonymous_numbered(self):
         eq_({'1': 'a', 'hi': 'test', '3': 'z'}, _btp(['hi=test', 'a', '3=z']))
 
     def test_unapproved_template(self):
-        document(title='Template:new').save()
+        TemplateDocumentFactory(title='Template:new')
```

Comment:
Er, nevermind. It doesn't do that already. It only picks a good default value that includes the prefix. I'll modify it to add the prefix.